proto/profile/tx.proto

```diff
@@ -21,3 +23,10 @@ message MsgCreateCoordinator {
 message MsgCreateCoordinatorResponse {
   uint64 coordinatorId = 1;
 }
+
+message MsgDeleteCoordinator {
+  string address = 1;
+}
+
+message MsgDeleteCoordinatorResponse {
```

Comment:
Here is the opposite of `MsgUpdateCoordinatorDescription`

It makes sense to add the `CoordinatorID` in the response because in the store this value is removed, we then communicate an information from a change in the store and that cannot be get afterward (`coordinator-from-addr` will not return the id)